lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -119,18 +119,26 @@ def ask_for_database_credentials
       self.database = just_ask("name of the database on #{host}", database) unless local?
       self.username = just_ask("username", username) unless local?
       count = 0
+      empty_count = 0
```

Comment:
original logic was to try 2 times and go to main menu after that, do you want to check only once ?